examples/getting_started/user_config.yaml.example

```diff
@@ -35,8 +35,6 @@ SIMULATION_CONTROLLER:
   TASK: "NullTask"
   CUSTOM_UNREAL_CONSOLE_COMMANDS: []
 
-  SCENE_ID: "debug_0000"
```

Comment:
Let's keep this explicitly set to `apartment_0000`. Even though this is not strictly necessary, it will highlight to the readers of `getting_started` that this is how the scene can be changed.